Sources/Network/Parse.swift

```diff
@@ -2,15 +2,17 @@
 //  Parse.swift
 //  Alicerce
 //
-//  Created by Luís Afonso on 06/04/2017.
+//  Created by Luís Portela on 06/04/2017.
 //  Copyright © 2017 Mindera. All rights reserved.
 //
 
 import UIKit
 
+public protocol ParseError: Swift.Error {}
```

Comment:
honestly, currently i don't know 🤔 as far as i can see it was only to have a base ParseError, but doesn't makes sense now. i'm not even using it in the app the i redefine the errors.

should remove it?